R/mod_export_data.R

```diff
@@ -13,341 +13,388 @@
 #' @importFrom data.table fwrite
 
 mod_export_data_ui <- function(id) {
-  ns <- NS(id)
-  tabPanel(
```

Comment:
Please align the number of spaces/tabs in the code with the previous version

Reply:
Fixed indentation in the latest commit